cmd/publishing-bot/config/config.go

```diff
@@ -18,6 +18,14 @@ package config
 
 // Config is how we are configured to talk to github.
 type Config struct {
+	// GithubHost is the address for github.
+	// Default to github.com
+	GithubHost string `yaml:"github-host"`
+
+	// Package is the package name for this repo.
+	// Default to k8s.io
+	Package string `yaml:"package"`
```

Comment:
Wouldn't something like BasePackage be better? We have many packages around. But this is the base of all of them.

Reply:
Updated to be `BasePackage`.